### ISSUE TITLE
@devmod/schematic:devmod-angular error

### ISSUE THREAD

**Issue by killbox-code** [NextFaze/devmod#9]
**Issue Type**

* [x ] Bug
* [ ] Feature Request
* [ ] Improvement

**Issue Description**

*existing project ng6 project*
```
yarn add @devmod/schematic
ng generate @devmod/schematic:devmod-angular
```

produces error:
```
Cannot read property 'architect' of undefined
```

My first thought was whether angular/cdk was installed - it was.

Started a fresh ng6 project and went through the steps again.. 

```
ng generate @devmod/schematic:devmod-angular
CREATE src/app/devmod.config.ts (396 bytes)
UPDATE src/app/app.component.html (1174 bytes)
UPDATE package.json (1437 bytes)
```

...works well so I'm going to try to dig into root cause if I get some time over the weekend. If I can come up with a repro I'll certainly pass it along but just wanted to submit an issue on it quickly for your tracking or if you've seen this before and can head-start me in the right direction.

Thanks for the great work guys, this tool is bad ass!

**Comment by zbarbuto** [comment]
Interesting. I can't say I've seen that error before but I've only really tried the schematics on a couple of internal apps and a blank cli project. Would be great if you could submit a simple repro project.

Can I ask was this in a single app project or a monorepo with multiple Angular apps in it?

**Comment by killbox-code** [comment]
It was against a monorepo with multiple angular apps whose structure follows the angular/cli v6 workspace feature. I've since backed off of that approach, for reasons not pertinent here, but upon trying again against a fresh monorepo everything works as desired. I even went back to the commit of my own repo on which I originally tried this and everything was fantastic.

It seems more likely at this point that I either had some momentary packages/project anomaly that was cleared up after clean/install or that I'd fat-fingered something. As of now, I've tried reproducing under almost a dozen different scenarios and they all come out integrating devmod via provided schematics.

Not a bad outcome - just means that this was more of a fire-drill due to unknown user/project-specific error never to be seen again.

If I come across anything to the contrary in the future I'll re-open but at this time I'll close this and drive on.
